rfc-138-enable-brotli-compression.md

```diff
@@ -0,0 +1,88 @@
+
+![Image of browser support for the brotli compression header.](rfc-138/brotli-usage-caniuse.png)
+
+This compression has been enabled on both integration and staging. We saw the following savings in terms of total page bytes for HTML, CSS, and JavaScript across the frontend applications of GOV.UK:
+
+| Page                               | Application             | Before (b) | After (b) | Diff (%) |
+|------------------------------------|-------------------------|------------|-----------|----------|
+| /                                  | frontend                | 259,371    | 248,653   | -4.13    |
+| /coronavirus                       | info-frontend           | 272,687    | 288,650   | -5.53    |
+| /browse/driving/driving-licences   | collections             | 278,385    | 265,832   | -4.51    |
+| /contact                           | feedback                | 242,047    | 232,124   | -4.10    |
+| /search/all                        | finder-frontend         | 313,400    | 295,979   | -5.56    |
+| /bank-holidays                     | frontend                | 263,686    | 251,555   | -4.60    |
+| /national-minimum-wage-rates       | government-frontend     | 266,699    | 254,073   | -4.73    |
+| /info/coronavirus/business-support | info-frontend           | 233,133    | 226,382   | -2.90    |
+| /licence-finder/sectors            | licence-finder          | 242,555    | 234,969   | -3.13    |
+| /guidance/immigration-rules        | manuals-frontend        | 250,431    | 240,763   | -3.86    |
+| /service-manual/service-standard   | service-manual-frontend | 254,529    | 245,881   | -3.40    |
+| /additional-commodity-code/y       | frontend                | 250,664    | 240,974   | -3.87    |
+| /government/people/theresa-may     | whitehall               | 277,888    | 265,642   | -4.41    |
+
+
+These savings for the pages tested converted into the following performance improvements. All tested on a simulated Moto G4 mobile on a 3G connection.
+
+### Homepage
+
+The visual progress of the homepage has improved by approximately 110 ms as can be seen in the visual progress graph:
+
+![Visual progress of the homepage using both compression methods.](rfc-138/visual-progress-homepage.png)
+
+And we see the a reduction in bytes for all the expected assets:
+
+![bytes for assets compared](rfc-138/total-bytes-homepage.png)
+
+### Coronavirus page
+
+The visual progress graph for this page starts off worse by 300ms, but quickly catches up and the viewport for Brotli completes rendering 200ms before gzip:
```

Comment:
Should this be "starts off worse by 400ms"?

Reply:
Yes! Thanks. Updated.